homeassistant/components/homekit/type_fans.py

```diff
@@ -127,9 +127,8 @@ def __init__(self, *args: Any) -> None:
                 self.preset_mode_chars[preset_mode] = preset_serv.configure_char(
                     CHAR_ON,
                     value=False,
-                    setter_callback=lambda value, preset_mode=preset_mode: self.set_preset_mode(
-                        value, preset_mode
-                    ),
+                    setter_callback=lambda value,
+                    preset_mode=preset_mode: self.set_preset_mode(value, preset_mode),
```

Comment:
We need to wrap the lambda in parenthesis, as multiline lambdas is not something we generally want as they are hard to read.